python/test/unit/tools/test_aot.py

```diff
@@ -99,19 +101,34 @@ def kernel(C, A, B, M, N, K,
 }"""
 
 
+def select_compiler():
+    gxx = shutil.which("g++")
+    icpx = shutil.which("icpx")
+    cl = shutil.which("cl")
+    cxx = icpx or cl if os.name == "nt" else icpx or gxx
```

Comment:
Nit: for humble developers who don't know what's is operators priority on Python)
```suggestion
    cxx = icpx or (cl if os.name == "nt" else icpx) or gxx
```

Reply:
I'll do it, but the brackets should be different: `cxx = (icpx or cl) if os.name == "nt" else (icpx or gxx)` :)

For example:
```python
>>> "a" or "b" if True else "c" or "d"
'a'
>>> None or "b" if True else "c" or "d"
'b'
>>> "a" or "b" if False else "c" or "d"
'c'
>>> "a" or "b" if False else None or "d"
'd'
```

Reply:
That shows we needed brackets even more) Thanks!